### PR TITLE
Visitor/User can see List of Recipes

### PR DESCRIPTION
**Pull request by beccaburns**
Visitor/User can see List of Recipes
==========================

[PT-Link](https://www.pivotaltracker.com/story/show/169667275)

## PR Content
In this PR, the user is able to view a list of recipes.

## In this PR
- requestRecipes module
- ListRecipes Component
- Axios GET request
- Recipe fixture file
- Integration test file

### CODE REVIEW

**Review by miyeaier — APPROVED** [review]

File: src/__tests__/listRecipes.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```
Comment (clalil) [inline src/__tests__/listRecipes.test.js:2]: Make sure this also passes if you keep it.

File: cypress/integration/userCanViewListOfRecipes.spec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+describe('FoodHub user can view a list of recipes', () => {
+  it('successfully', () => {
+    cy.server()
+    cy.route({
```
Comment (clalil) [inline cypress/integration/userCanViewListOfRecipes.spec.js:9]: Please add sad path test.

File: src/components/ListRecipes.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { getData } from '../modules/requestRecipes';
+import { Container } from 'semantic-ui-react';
+
+class ListRecipes extends Component {
+  state = {
+    recipes: [],
+    error_message: null
```
Comment (tochman) [inline src/components/ListRecipes.jsx]: what is `error_message` doing for us?

File: src/components/ListRecipes.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  }
+
+  componentDidMount() {
+    this.getRecipes()
+  }
+
+  async getRecipes() {
+    let result = await getData()
```
Comment (tochman) [inline src/components/ListRecipes.jsx]: `getData` or `getReciepes`?

File: src/components/ListRecipes.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  }
+
+  render() {
+    let renderListRecipes
+    const recipeData = this.state.recipes
+
+    if (recipeData !== []) {
+      renderListRecipes = recipeData.map(recipe => {
+        return (
+          <Container>
```
Comment (tochman) [inline src/components/ListRecipes.jsx]: Is a set of `containers` really needed here?

File: cypress.json [hunk 6]
```diff
@@ -1 +1,7 @@
-{}
+{
+  "video": false,
+  "viewportWidth": 900,
+  "viewportHeight": 700,
+  "autoOpenDevtoolsForTabs": true,
```
Comment (tochman) [inline cypress.json]: I added this setting but it does nothing for us. Please remove it.

**Pull request merged**